.travis.yml

```diff
@@ -19,13 +19,15 @@ compiler:
 
 before_install:
   - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
+  - sudo apt-add-repository -y ppa:mutlaqja/ppa
```

Comment:
Travis uses "Ubuntu 14.04.5 LTS". INDI needs 15.10 or higher [1].

You'll have to install it from source [2,3] and check which dependencies would you need to compile first.

[1] https://launchpad.net/~mutlaqja/+archive/ubuntu/ppa/+packages?batch=75
[2] https://github.com/indilib/indi
[3] https://github.com/indilib/indi/releases/tag/v1.5.0